src/gtest/README.md

```diff
@@ -0,0 +1,72 @@
+## Valkey Google Unit Test Framework
+This directory contains the Google Test (gtest) framework integration for
+Valkey unit testing. The framework uses Google Test and Google Mock (gmock)
```

Comment:
You don't need to mention GoogleMock.  In the past, these were 2 separate projects.  But now, it's all considered "GoogleTest" (or "gtest" for short)